scripts/auto-cpufreq-runit

```diff
@@ -1,2 +1,14 @@
 #!/bin/sh
-exec /usr/local/bin/auto-cpufreq --daemon
+# test for update
+if git -c advice.detachedHead=false pull -v https://github.com/AdnanHodzic/auto-cpufreq.git | grep -q -v '^\s*\(Updating\|Already up-to-date\|From\s\)\s'; then
```

Comment:
There is no guarantee that `git` exists. The script should check for dependencies like `git` and `sudo` mentioned below and act approprietly.

Where is `git` pulling to exactly?